site/js/main.js

```diff
@@ -174,15 +174,20 @@ function updateThresholdCalculation() {
 		var silver = Math.floor(item.price / 100) % 100;
 		var copper = item.price % 100;
 		list += '<div class="entry" id="list-entry-' + item.section + '-' + item.details.id + '"><div class="entry-checkbox"><input type="checkbox" /></div>'
-			  + '<div class="entry-name">' + item.name + '</div><div class="entry-price"><span>' + gold + '</span><span class="base-icon gold-icon" role="img" aria-label="Gold"></span>'
-		      + ' <span id="min-unlock-silver">' + silver + '</span><span class="base-icon silver-icon" role="img" aria-label="Silver"></span>'
-			  + ' <span id="min-unlock-copper">' + copper + '</span><span class="base-icon copper-icon" role="img" aria-label="Copper"></span></div></div>';
-	}	
-	
+			  + '<div class="entry-name">' + item.name +'</div><div class="entry-name">' + capitalizeFirstLetter(item.section) + '</div><div class="entry-price"><span>' + gold + '</span><span class="base-icon gold-icon" role="img" aria-label="Gold"></span>'
```

Comment:
I think this is the one thing that gives me pause. ```item.section``` is technically the id of the section, where a display name for the section is captured in metadata - although not as a mapping. Theoretically we could pull out that mapping as a lookup and get the display from there. This would mean that, for example, the label for a back piece would be "Back Items" rather than "Back"... which still isn't great because of the plural I guess.

Reply:
Yea I knew that I'm misusing the section property for this..
However in my testing with an almost empty account this looks fine, only the Back-Items are the only exception beeing listed with only using the term "Back"

I was mostly interessted in seeing this for dyes as the pure name doen't always imply that it is a dye.

I think adding the singular name to the config would be best. The pluar form just won't sound right for a single item.
I'd add this as a field called "type" or "name_singular" to each section.

Reply:
I like the idea of adding an extra field for it, provided through the config. :)